applications/StructuralMechanicsApplication/tests/test_patch_test_shells.py

```diff
@@ -0,0 +1,229 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics 
+
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+
+class TestPatchTestShells(KratosUnittest.TestCase):
+    def setUp(self):
+        pass
+    
+
+    def _add_variables(self,mp):
+        mp.AddNodalSolutionStepVariable(KratosMultiphysics.DISPLACEMENT)
+        mp.AddNodalSolutionStepVariable(KratosMultiphysics.ROTATION)
+        mp.AddNodalSolutionStepVariable(KratosMultiphysics.REACTION)
+        mp.AddNodalSolutionStepVariable(KratosMultiphysics.VOLUME_ACCELERATION)  
+        mp.AddNodalSolutionStepVariable(StructuralMechanicsApplication.POINT_LOAD)      
+        
+    
+    def _add_dofs(self,mp):
+        for node in mp.Nodes:
```

Comment:
There is a fjnction in variable utils to add dofs in parallel. Use rhat if you can.

Also  associate a reaction to the rotstion

Reply:
Now it works :+1: